http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BaseRuntimeInterpreter.scala

```diff
@@ -209,7 +213,8 @@ abstract private[util] class BaseRuntimeInterpreter[F[_]](
   }
 
   // Shutdown script to run after the runtime is paused
-  protected def getShutdownScript(runtimeAndRuntimeConfig: RuntimeAndRuntimeConfig): F[Map[String, String]] = {
+  protected def getShutdownScript(runtimeAndRuntimeConfig: RuntimeAndRuntimeConfig,
+                                  shouldDeleteJupyterDir: Boolean): F[Map[String, String]] = {
```

Comment:
what exactly is the use case for this flag?

Reply:
The flag is to tell whether shutdown script should delete `.jupyter` dir during shutting down. It's only true when it's GCE VM and user wants to delete the VM.

It'll have issue if we create runtimes with different images using the same PD. Will update RuntimeCreationDiskSpec to  verify that